extensions/blocks/tiled-gallery/gallery-image/edit.js

```diff
@@ -121,16 +131,33 @@ class GalleryImageEdit extends Component {
 					[ `filter__${ imageFilter }` ]: !! imageFilter,
 				} ) }
 			>
-				{ isSelected && (
-					<div className="tiled-gallery__item__inline-menu">
-						<IconButton
-							icon="no-alt"
-							onClick={ onRemove }
-							className="tiled-gallery__item__remove"
-							label={ __( 'Remove Image', 'jetpack' ) }
-						/>
-					</div>
-				) }
+				<div className="tiled-gallery__item__move-menu">
+					<IconButton
```

Comment:
Deprecated `<IconButton>` can be changed to `<Button icon>` once it's available in WP.